lib/optimizely/index.js

```diff
@@ -110,11 +110,13 @@ Optimizely.prototype.roots = function(){
   if (!data) return;
   var allExperiments = data.experiments;
   if (!data || !data.state || !allExperiments) return;
-  var variationNamesMap = data.state.variationNamesMap;
-  var variationIdsMap = data.state.variationIdsMap;
-  var activeExperimentIds = data.state.activeExperiments;
-  var activeExperiments = getExperiments(activeExperimentIds, variationNamesMap,
-    variationIdsMap, allExperiments);
+  var options = {
+    variationNamesMap: data.state.variationNamesMap,
+    variationIdsMap: data.state.variationIdsMap,
+    activeExperimentIds: data.state.activeExperiments,
+    allExperiments: allExperiments
+  };
+  var activeExperiments = getExperiments(options);
```

Comment:
If we don't use `options` anywhere else, no need to store it in a variable—just pass it directly to `getExperiments`